frontend/internal-packages/schema-bench/src/cli/executeOpenaiUnified.ts

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env node
+
+import { resolve } from 'node:path'
+import { config as loadEnv } from 'dotenv'
+import { err, ok, type Result } from 'neverthrow'
+import * as v from 'valibot'
+import { OpenAIExecutor } from '../executors/openai/openaiExecutor'
+import type { OpenAIExecutorInput } from '../executors/openai/types'
+import {
+  filterAndResolveDatasets,
+  getWorkspacePath,
+  handleCliError,
+  handleUnexpectedError,
+  loadInputFiles,
+  parseArgs,
+  saveOutputFile,
+  selectTargetDatasets,
+} from './utils'
+
+// parseArgs moved to ./utils
```

Comment:
Comments explaining the background should be removed as they make maintenance difficult.